cmd/argocd/commands/app.go

```diff
@@ -507,6 +537,7 @@ func addAppFlags(command *cobra.Command, opts *appOptions) {
 	command.Flags().StringVar(&opts.namePrefix, "nameprefix", "", "Kustomize nameprefix")
 	command.Flags().BoolVar(&opts.directoryRecurse, "directory-recurse", false, "Recurse directory")
 	command.Flags().StringVar(&opts.configManagementPlugin, "config-management-plugin", "", "Config management plugin name")
+	command.Flags().StringArrayVar(&opts.tlaParameters, "tla-parameters", []string{}, "Jsonnet top level arguments")
```

Comment:
hmm, I wonder if we should prefix this wish `--jsonnet-`? e.g. `--jsonnet-tlas`?

Reply:
I will go with  jsonnet-tlas, I think it makes the best connection with what they mean